src/tests/system/tests/test_sssctl.py

```diff
@@ -1078,8 +1079,8 @@ def test_sssctl__analyze_tevent_id(client: Client, ldap: LDAP):
 
 @pytest.mark.tools
 @pytest.mark.ticket(bz=2013260)
-@pytest.mark.topology(KnownTopology.LDAP)
-def test_sssctl__analyze_child_logs(client: Client, ldap: LDAP):
+@pytest.mark.topology(KnownTopology.IPA)
+def test_sssctl__analyze_child_logs(client: Client, ipa: IPA):
     """
     :title: "sssctl analyze" to parse child logs
```

Comment:
Why is this test limited to the IPA provider? Seems like this could also work for AD/Samba?

Also can you provide a small :description: summarizing the bug, hopefully it will prevent maintainers needing to dive into each bz. From what I can glean it looks like 

:description: Login failures did not log 'Preauthentication' failure messages correctly for sssctl to analyze

Reply:
>> Why is this test limited to the IPA provider? Seems like this could also work for AD/Samba?

Yes it could, it just doesn't seem worthwhile  running it against more than one provider, in my mind the test itself is quite low value/importance.

>> Also can you provide a small :description: summarizing the bug,

It is not related to a bug, just functionality that was added as part of the analyzer feature. I added a short description.